apps/storybook/stories/Namekit/CurrencySymbol.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import { Currency } from "@namehash/ens-utils";
+import { PriceDisplaySize, CurrencySymbol } from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+export const ETH: Story = {
+  args: {
+    currency: Currency.Eth,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const USD: Story = {
+  args: {
+    currency: Currency.Usd,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const USDC: Story = {
+  args: {
+    currency: Currency.Usdc,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const WETH: Story = {
+  args: {
+    currency: Currency.Weth,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const DAI: Story = {
+  args: {
+    currency: Currency.Dai,
+    size: PriceDisplaySize.Large,
+  },
+};
+export const MicroSize: Story = {
+  args: {
+    size: PriceDisplaySize.Micro,
+    currency: Currency.Eth,
+  },
+};
+export const SmallSize: Story = {
+  args: {
+    size: PriceDisplaySize.Small,
+    currency: Currency.Eth,
+  },
+};
+export const MediumSize: Story = {
+  args: {
+    size: PriceDisplaySize.Medium,
+    currency: Currency.Eth,
+  },
+};
+export const LargeSize: Story = {
```

Comment:
MediumSize and LargeSize appear exactly the same to me on my laptop.

Reply:
These in fact are differently sized both in Storybook and Figma:

Reply:
https://github.com/namehash/nameguard/assets/49823133/675cff4f-b5c9-4aac-8167-25870ebfe189

Reply:
@FrancoAguzzi Yes because you're looking at a `DisplayedPrice` here, but my feedback above was for a `CurrencySymbol`.

Maybe there actually is a difference but only for the font and only for the currency symbol `$` which needs to match the font of the `DisplayedPrice`?

Reply:
Ah, true!


Correct! This is the PriceDisplaySize mapping to classname we currently have:

![Captura de Tela 2024-07-11 às 22 06 11](https://github.com/user-attachments/assets/ce8efea4-0781-48b2-9375-b0c9980061c4)

Note how micro and small are the same size as medium and large are,

Key references to help rethink sizes: Figma [artboards](https://www.figma.com/design/2e3p7KEFikJjTPXdRyOWfZ/Namekit?node-id=6289-10398&m=dev) and size mapping shared above :)

Reply:
@FrancoAguzzi Thanks. Please see my related comment about introducing a `CurrencySymbolSize` enum.

Reply:
@FrancoAguzzi This is still an open question on this PR.